WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -563,6 +569,9 @@ - (void)setEditing:(BOOL)editing animated:(BOOL)animated
     if (editing) {
         [self updateHeaderSize];
         self.tableView.tableHeaderView = self.headerView;
+
+        self.firstDisable = 0;
```

Comment:
Assigning an integer to an `NSDate` looks a bit odd. We prefer to be very consistent with types and assignments.  It might be better to call `self.firstDisable = [NSDate date];` here instead of the conditional in `visibilitySwitchAction`.